nucleus/rna/models.py

```diff
@@ -35,14 +35,14 @@ class Release(SaveToGithubModel):
                 'Firefox Extended Support Release', 'Firefox OS',
                 'Thunderbird', 'Firefox for iOS')
 
+    is_public = models.BooleanField(default=False, help_text = "Note: If checked, these Release Notes will be visible on www.mozilla.org")
```

Comment:
Unfortunately the order here doesn't affect the order on the admin page. We need to define the list of fields in the [`fields` param of the admin class](https://docs.djangoproject.com/en/2.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.fields).

Reply:
I should say I don't think it reliably affects the order, the way to make sure is in the docs I liked above.

Reply:
I tried locally and it works for me:
![image](https://user-images.githubusercontent.com/454375/135415828-efe9d658-b260-40a7-bddf-2930c69a34cf.png)

Reply:
huh. cool. well maybe that's fine then.